Makefile

```diff
@@ -200,13 +200,18 @@ $(RESOURCES_DOT_GO): $(NPM)
 	rm -Rf dist && \
 	gulp build && \
 	cd - && \
-	rm -f bin/tarfs bin/rsrc && \
+	rm -f bin/tarfs && \
 	go install github.com/getlantern/tarfs/tarfs && \
+	mv bin/tarfs.exe bin/tarfs ; \
 	echo "// +build !stub" > $$DEST && \
 	echo " " >> $$DEST && \
-	tarfs -pkg ui $$DIST >> $$DEST && \
+	bin/tarfs -pkg ui $$DIST >> $$DEST
+
+# Generates a syso file that embeds an icon for the Windows executable
+generate-windows-icon:
```

Comment:
The syso is now back in git so I don't need to run this on Windows, which is good because it doesn't run.